easybuild/easyconfigs/b/Bazel/Bazel-3.7.2-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,37 @@
+name = 'Bazel'
+version = '3.7.2'
+
+homepage = 'https://bazel.io/'
+description = """Bazel is a build tool that builds code quickly and reliably.
+It is used to build the majority of Google's software."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/bazelbuild/%(namelower)s/releases/download/%(version)s']
+sources = ['%(namelower)s-%(version)s-dist.zip']
+patches = [
+    'Bazel-3.4.1-fix-grpc-protoc.patch',
+    'Bazel-3.7.1_fix-protobuf-env.patch',
+    'Bazel-3.7.2_fix-GCC-11.patch',
+    'Bazel-3.7.2_fix-Java-15.patch',
+]
+checksums = [
+    'de255bb42163a915312df9f4b86e5b874b46d9e8d4b72604b5123c3a845ed9b1',  # bazel-3.7.2-dist.zip
+    'f87ad8ad6922fd9c974381ea22b7b0e6502ccad5e532145f179b80d5599e24ac',  # Bazel-3.4.1-fix-grpc-protoc.patch
+    '8706ecc99b658e0a96c38dc2c23e44da35059b85f308602aac76a6d6680376e7',  # Bazel-3.7.1_fix-protobuf-env.patch
+    '7fd5846f2ed1f27af9528a900f284f95ad982f3e7d486264c3bdee890a9c22fb',  # Bazel-3.7.2_fix-GCC-11.patch
+    '28f394f561824abf562556877483bc443dfd3ee0c7cba48a56e22670258669e7',  # Bazel-3.7.2_fix-Java-15.patch
+]
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('Python', '3.9.6'),
+    ('Zip', '3.0'),
+]
+
+dependencies = [('Java', '15', '', True)]
```

Comment:
Note: this was downgraded to `Java/11` in #14734, since we had already picked a Java version for the `2021b` generation of easyconfigs (in `R-4.1.2-foss-2021b.eb` for example).

That change was needed to avoid trouble with other easyconfigs using `GCCcore/11.2.0`, see for example `canu` in #14724